pkg/csi/service/wcp/controller.go

```diff
@@ -497,16 +493,16 @@ func (c *controller) createBlockVolume(ctx context.Context, req *csi.CreateVolum
 			}
 		} else {
 			// zone labels not Present in the topologyRequirement
```

Comment:
"// zone labels not Present in the topologyRequirement"

Is this for non-stretched? because will always get zone and host both for vdpp workload on stretched supervisor and only host in non-stretched supervisor cluster 


and if we are getting both host and zone why don't we directly look up datastore in the reqeusted zone and host.

Reply:
I agree, this code was originally written for single cluster supervisors and will only get implemented when just hostname is passed in topology requirement. Why are we changing this part of the code?